vue.config.js

```diff
@@ -23,7 +23,11 @@ module.exports = {
   publicPath: '',
   outputDir: 'dist',
   indexPath: 'index.html',
-  transpileDependencies: ['vuetify'],
+  transpileDependencies: [
+    'graphql-language-service-interface',
+    'graphql-language-service-parser',
```

Comment:
These two dependencies broke the build due to the null-coallesce operator. I think that could happen in our current Yarn V1 build too at some point?